adb-vagrantfile-tests/adb-release-mesos/test-new-build-mesos.py

```diff
@@ -0,0 +1,115 @@
+#!/usr/bin/python
+
+import os
+import unittest
+import subprocess
+
+from unittest import TestCase
+
+VAGRANTFILE_PATH = \
+    "/root/adb/components/centos/centos-mesos-marathon-singlenode-setup/" \
+    "Vagrantfile"
+
+CWD = "/root/adb/components/centos/centos-mesos-marathon-singlenode-setup/"
```

Comment:
I was thinking if there is a better way to provide this path than hardcoding it.